src/installer/installer.go

```diff
@@ -153,7 +151,7 @@ func (i *installer) updateSingleNodeIgnition(singleNodeIgnitionPath string) erro
 	}
 	// TODO: update this once we can get the full host specific overrides we have in the ignition
 	// Remove the Config part since we only want the rest of the overrides
-	hostConfig.Ignition.Config = config31types.IgnitionConfig{}
+	hostConfig.Ignition.Config = ignition.EmptyIgnitionConfig
```

Comment:
Hmmm, seems like this should be the master ignition we get from the service no?

Reply:
why?

Reply:
if i understand right the comment, we just remove all the host specific ignition part